### ISSUE TITLE
demo banner: "view demo details" link doesn't have focus state

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#180]
Applies to all demos that are self contained with bar above, such as https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/fishbowlie/

Steps:

Tab to view demo details

Expected:

Should have focus rect

Actual:

Nadda
